SparkPost/support-docs
# WE-166 Single Page Template

## What Changed

- Added new DocumentationPage template
- Upgrade Matchbox to 6.1.0

## How to verify
- Run locally or visit deploy link and visit any momentum page.
- To verify the last updated styled go to `pages/momentum/[slug].tsx` and pass in a random string like `September 2021` to the `lastUpdated` prop
